### PR TITLE
Rke2 Patches 09/23/2021

### PR DESCRIPTION
**Pull request by dereknola**
Signed-off-by: dereknola <derek.nola@suse.com>

https://github.com/rancher/rke2/issues/1809
https://github.com/rancher/rke2/issues/1810
https://github.com/rancher/rke2/issues/1861

### CODE REVIEW

**Review by kinarashah — DISMISSED** [review]

File: channels-rke2.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  - version: v1.19.14+rke2r1
+  - version: v1.19.15+rke2r1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.20.10+rke2r1
+  - version: v1.20.11+rke2r1
     minChannelServerVersion: v2.6.0-alpha1
     maxChannelServerVersion: v2.6.99
-  - version: v1.21.4+rke2r2
+  - version: v1.21.5+rke2r2
```
Comment (deniseschannon) [inline channels-rke2.yaml]: This is not how you update a fully released RKE2.

**Review by deniseschannon — CHANGES_REQUESTED** [review]

File: channels-rke2.yaml (hunk 1 above)
Comment (dereknola) [reply channels-rke2.yaml]: Apologies on the that, will fix.

File: channels-rke2.yaml [hunk 2]
```diff
@@ -116,3 +116,9 @@ releases:
       rancher-vsphere-csi:
         repo: rancher-charts
         version: 100.0.0
+  - version: v1.21.5+rke2r1
```
Comment (deniseschannon) [inline channels-rke2.yaml:119]: Have we confirmed that the charts packaged in v1.21.5 will be the same?

How is this going to be different from what is being introduced in dev-v.6 (aka Rancher 2.6.1) 

Will there be a v1.21.5+rke2r2 shipping with 2.6.1? 

https://github.com/rancher/kontainer-driver-metadata/blob/dev-v2.6/channels-rke2.yaml#L119

Comment (brandond) [reply channels-rke2.yaml:119]: I'm also confused - why is this going into this specific target branch, and why is it missing all the stuff that's currently in dev-v2.6 as linked above by Denise? We already have `serverArgs-v2`, `agentArgs-v2`, `charts-v2` and this is missing all of that.

Comment (deniseschannon) [reply channels-rke2.yaml:119]: It's on a separate target branch as we wanted to get these k8s versions out and available due to the security fixes in it and not wait until 2.6.1 for the updates to be released. RKE1 was already released from this branch into production.

Comment (deniseschannon) [reply channels-rke2.yaml:119]: Due to the lack of support in Harvester, the min Rancher version will need be changed to 2.6.1 as well.

**Comment by dereknola** [comment]
Closing after team discussion.

**Pull request closed without merging**